### PR TITLE
[stable/postgresql] Allow initDb scripts in the form of secrets

### LINKED ISSUE #1044: Postgresql - How to manage credentials.
I can't seem to find a decent way to manage my credentials while leveraging the Postgresql Helm chart. I see there are various options for specifying initdbScripts, which is where we setup our usernames/credentials, but I don't know which one to use. 

Currently, I use the Configmap option, but ConfigMaps are not considered secure in Kubernetes. Would it be possible to add a similar option that leverages a Secret instead? 

As for the other options, as far as I can tell, I would have to modify this chart in order to mount the scripts into the initdb directory, which isn't ideal. The third option is putting the scripts in the values.yaml file, but putting all of our schema scripts there gets out of hand/ugly very quickly.

Does anyone have a suggestion on how to manage our database credentials with this chart? Thanks.

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@bitnami.com>

#### What this PR does / why we need it:
Some initialization scripts may contain sensitive data such as passwords or credentials. This PR enables to use an external secret with these initialization scripts.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes https://github.com/bitnami/charts/issues/1044#issuecomment-461092528

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/postgresql/templates/statefulset.yaml [hunk 1]
```diff
@@ -185,7 +185,11 @@ spec:
         volumeMounts:
         {{- if or (.Files.Glob "files/docker-entrypoint-initdb.d/*.{sh,sql,sql.gz}") .Values.initdbScriptsConfigMap .Values.initdbScripts }}
         - name: custom-init-scripts
-          mountPath: /docker-entrypoint-initdb.d
+          mountPath: /docker-entrypoint-initdb.d/configmap
```
Comment (tompizmor) [inline stable/postgresql/templates/statefulset.yaml:188]: can you mount some scripts at `configmap` folder, others at `secret` folder, and postgresql will take all of them?

**Review by tompizmor — COMMENTED** [review]
/ok-to-test

**Comment by tompizmor** [comment]
@javsalgar resolve the conflicts

File: stable/postgresql/templates/statefulset.yaml (hunk 1 above)
Comment (javsalgar) [reply stable/postgresql/templates/statefulset.yaml:188]: Yes, as you can see here https://github.com/bitnami/bitnami-docker-postgresql/blob/master/9.6/debian-9/rootfs/run.sh

Comment (tompizmor) [reply stable/postgresql/templates/statefulset.yaml:188]: ok!

**Review by tompizmor — APPROVED** [review]
/lgtm

**Pull request merged**